website/content/en/highlights/2022-08-08-internal-metrics-cardinality.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Sorry, I probably wasn't clear enough. I was thinking this could actually just go in the upgrade guide for 0.24: https://github.com/vectordotdev/vector/blob/master/website/content/en/highlights/2022-08-16-0-24-0-upgrade-guide.md